pkg/releaseutil/manifest.go

```diff
@@ -31,17 +31,19 @@ type SimpleHead struct {
 	} `json:"metadata,omitempty"`
 }
 
+var sep = regexp.MustCompile("(?:^|\\s*\n)---\\s*")
```

Comment:
I intend to add these boundary markers (both starting and ending) in a fix for #2181.